### PR TITLE
SorrTask 481 Unit test plot_cols()

### PR DESCRIPTION
**Pull request by zli00185**
#481

### CODE REVIEW

**Comment by zli00185** [comment]
The function and data was confirm workable. Here is the screenshot.
Screenshot 2023-08-16 at 8 47 33 PM

File: core/plotting/test/test_plots.py [hunk 1]
```diff
@@ -29,6 +29,20 @@ def get_plot_spectrum1() -> pd.DataFrame:
         df = pd.DataFrame(data)
         return df
 
+    @staticmethod
+    def get_plot_col1() -> pd.DataFrame:
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:33]: - `get_plot_col1()` and `get_plot_effective_correlation_rank1()` are identaica
- seems like `get_plot_heatmap1()` and `get_plot_spectrum1()` can be easily replaced by  `get_plot_col1()` as well

So my proposal is
- create `get_test_plot_df()` which is a copy of this func
- deprecate all the functions mentioned above and replace their usage with `get_test_plot_df()`
- rerun test and notebook, make sure everything passes and looks good

File: core/plotting/test/test_plots.py [hunk 2]
```diff
@@ -170,3 +184,10 @@ def test_plot_spectrum2(self) -> None:
         _, axes = plt.subplots(2, 2, figsize=(20, 10))
         axes_flat = axes.flatten()
         cplmiplo.plot_spectrum(signal=test_df, axes=axes_flat)
+
+    def test_plot_cols(self) -> None:
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:188]: -> `test_plot_cols1`

**Review by DanilYachmenev — COMMENTED** [review]
lg, see the further suggestions

File: core/plotting/test/test_plots.py (hunk 1 above)
Comment (zli00185) [reply core/plotting/test/test_plots.py:33]: Should I make the function public?

**Comment by zli00185** [comment]
All unit test pass
Screenshot 2023-08-17 at 11 01 22 AM

File: core/plotting/test/test_plots.py [hunk 3]
```diff
@@ -18,31 +18,20 @@ class Test_plots(unittest.TestCase):
     """
 
     @staticmethod
-    def get_plot_spectrum1() -> pd.DataFrame:
+    def get_test_plot_df() -> pd.DataFrame:
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:21]: for all test methods we finish their names with a respective integers
so here it should be `get_test_plot_df1`

File: core/plotting/test/test_plots.py [hunk 4]
```diff
@@ -18,31 +18,20 @@ class Test_plots(unittest.TestCase):
...
         """
-        Generate a test DataFrame for the plot_spectrum function.
+        Generate a test DataFrame for the plot_cols, plot_spectrum,
+        plot_effective_correlation_rank function.
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:24]: `Generate a DataFrame with random values to test plotting functions.`

File: core/plotting/test/test_plots.py (hunk 3 above)
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:21]: replace `get_plot_heatmap1()` with this one as well

Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:21]: And now lets also make replacements in the gallery notebook
you deprecate the functions so make sure you update all their calls

check that test for the gallery notebook passes

**Comment by zli00185** [comment]
Not sure what cause the test fail. I ran the purest and all passed

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

test failures are not related to the PR changes

**Pull request merged**